### PR TITLE
[SLDEV-2222] - use pom-integration from infra

### PR DESCRIPTION
**Pull request by RoniSegal**

### CODE REVIEW

File: src/main/java/io/sealights/plugins/sealightsjenkins/integration/JenkinsPomFile.java [hunk 1]
```diff
@@ -48,4 +50,15 @@ protected Document getDocumentInternal() throws IOException, InterruptedExceptio
         Document doc = fp.act(new GetDocumentFileCallable());
         return doc;
     }
+
+    @Override
+    //TODO implement
+    public void backup() throws Exception {
+        String backupFile = this.filename + BACKUP_EXTENSION;
+        log.info("JenkinsPomFile - creating a back up file: " + backupFile);
+        VirtualChannel channel = Computer.currentComputer().getChannel();
```
Comment (nadavye) [inline src/main/java/io/sealights/plugins/sealightsjenkins/integration/JenkinsPomFile.java]: Should we clean that using the CleanupManager?

File: src/test/java/io/sealights/plugins/sealightsjenkins/integration/MavenIntegrationTest.java [hunk 2]
```diff
@@ -21,281 +21,281 @@
...
+//
+//    private String getTestFolder(String testCaseName) {
+//        return PathUtils.join(PATH, testCaseName);
+//    }
+//
+//    private class MavenIntegrationMock extends MavenIntegration{
+//
+//        public MavenIntegrationMock(Logger log, MavenIntegrationInfo mavenIntegrationInfo, boolean isJenkinsEnvironment) {
+//            super(log, mavenIntegrationInfo, isJenkinsEnvironment);
+//        }
```
Comment (nadavye) [inline src/test/java/io/sealights/plugins/sealightsjenkins/integration/MavenIntegrationTest.java]: Do we need this code?

File: src/main/java/io/sealights/plugins/sealightsjenkins/integration/JenkinsPomFile.java (hunk 1 above)
Comment (RoniSegal) [reply src/main/java/io/sealights/plugins/sealightsjenkins/integration/JenkinsPomFile.java]: yup, and it cleaned outside

File: src/test/java/io/sealights/plugins/sealightsjenkins/integration/MavenIntegrationTest.java (hunk 2 above)
Comment (RoniSegal) [reply src/test/java/io/sealights/plugins/sealightsjenkins/integration/MavenIntegrationTest.java]: done

**Pull request merged**